packages/terminal/src/browser/terminal-widget-impl.ts

```diff
@@ -115,6 +115,17 @@ export class TerminalWidgetImpl extends TerminalWidget implements StatefulWidget
             });
         }));
 
+        this.term.attachCustomKeyEventHandler(event => {
+            const ctrlCmd = event.ctrlKey || event.metaKey;
+            const key = event.key;
+            if (ctrlCmd && key === 'c' && this.term.hasSelection()) {
```

Comment:
Hi, you should not use event.key - it's work only for English keyboard layout. This keyHandler should cove all keyboard layout, so use `event.keyCode()`
`c = keycode 67`
`v = keycode 86`

Reply:
There is some namespace doing some [name/keycode conversion](https://github.com/theia-ide/theia/blob/760b05426fc5590adef7cfb4b105f7c7b0273e29/packages/core/src/browser/keys.ts#L547) if you need it :)

Reply:
Thanks @marechal-p i'll check it out. @AndrienkoAleksandr afaik, keyCode has long been deprecated and is being dropped from newer browsers?  If that's not the case then ill just use the namespaced conversion given above. Thanks again for the input.

Reply:
@AndrienkoAleksandr actually, it would seem that the most layout agnostic value is `event.code`, which returns a string name thing: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code

`event.keyCode` is indeed deprecated, and `event.key` is layout-dependent.